pact3.0/03-coin-interface.repl

```diff
@@ -0,0 +1,83 @@
+; --------------------------------------------------------------------------
+; --------------------------------------------------------------------------
+; Using formal verification in Pact interfaces to create sample coin contract.
+
+
+; --------------------------------------------------------------------------
+; Installs account schema module
+; --------------------------------------------------------------------------
+(begin-tx)
+(env-data { "admin-keyset": ["product-dept" "dev-dept"] })
+
+(module acct-module ACCT-GOVERNANCE
+  @doc "account schema module"
+
+  (defcap ACCT-GOVERNANCE ()
+    (enforce-guard (read-keyset "admin-keyset")))
```

Comment:
Actually for this example don't do the delegated governance stuff, just use `(defcap GOVERNANCE () true)`